src/rdb.c

```diff
@@ -1450,6 +1446,7 @@ int rdbSaveBackground(char *filename, rdbSaveInfo *rsi) {
             return C_ERR;
         }
         serverLog(LL_NOTICE,"Background saving started by pid %ld",(long) childpid);
+        server.stat_keys_on_bgsave_start = dbTotalServerKeyCount();
```

Comment:
i think we want to do this in rewriteAppendOnlyFileBackground too.
like we show COW for all of these background operations that cycle thought the entire keyspace, i think we can show "progress" of key count for all of these.

if we rename the `bgsave` part of that variable to `save` i think it applies nicely to AOFRW too (a type of a save)

Reply:
Done